scrapers/pantyjobs.yml

```
@@ -0,0 +1,25 @@
+name: pantyjobs
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - pantyjobs.com/scenes/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: '//div[@class=''col-md-12 aligncenter'']/h1'
+      Performers:
+        Name: '//h2[1]'
+      Details:
+        selector: >-
+          //div[@class="post-entry"]/p[not (starts-with(.,"Members get full
+          access"))]
+        concat: |+
+
+
```

You are using a combo of the concat function and yaml's block chomping indicator
While it seems to work it is less readable  and prone to accidental removal of the blank line after the concat (and thus failure) compared to a simple 
```suggestion
        concat: "\n\n"
```